test/ANNOTATIONS.yaml

```diff
@@ -367,6 +373,10 @@ nbody:
     - chap04 Subtest random glitch
   01/17/16:
     - add dead string literal elimination (#3120)
+  03/14/16:
+    - upgrade jemalloc to 4.1.0 (#3447)
+  03/16/16:
+    - revert jemalloc to 4.0.4 (#3477)
```

Comment:
Hmm it was definitely merged with https://github.com/chapel-lang/chapel/pull/3477. I would guess either SF is down, or maybe it's getting marked as spam because it has a bunch off files changed?